ci/symfony.yml

```diff
@@ -0,0 +1,32 @@
+name: Symfony
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  symfony-tests:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Install Dependencies
+      run: composer install -q --no-ansi --no-interaction --no-scripts --no-suggest --no-progress --prefer-dist
```

Comment:
When I try this I get 
```

Run composer install -q --no-ansi --no-interaction --no-scripts --no-suggest --no-progress --prefer-dist
  composer install -q --no-ansi --no-interaction --no-scripts --no-suggest --no-progress --prefer-dist
  shell: /bin/bash -e {0}
##[error]Process completed with exit code 1.
```

Suggesting that composer isn't present, where would a user get composer from?